airbyte-integrations/connectors/source-retently/source_retently/schemas/feedback.json

```diff
@@ -0,0 +1,48 @@
+{
+  "type": "object",
+  "properties": {
+      "id": { "type": "string" },
+      "customerId": { "type": "string" },
+      "email": { "type": "string" },
+      "firstName": { "type": "string" },
+      "lastName": { "type": "string" },
+      "companyName": { "type": "string" },
+      "jobTitle": { "type": "string" },
+      "country": { "type": "string" },
+      "state": { "type": "string" },
+      "city": { "type": "string" },
+      "tags": {
+        "type": "array",
+        "items": { "type": "string" }
+      },
+      "customProps": {
+         "type": "array",
+         "items": { "type": "string" }
+       },
+      "campaignId": { "type": "string" },
+      "campaignName": { "type": "string" },
+      "createdDate": { "type": "string" },
+      "score": { "type": "number" },
+      "comment": { "type": "string" },
+      "checkbox": { "type": "boolean" },
+      "additionalQuestions": {
+        "type": "array",
+        "items": { "type": "string" }
+      },
+      "feedbackTags":  {
+        "type": "array",
+        "items": { "type": "string" }
+      },
+      "notes": {
+        "type": "array",
+        "items": { "type": "string" }
+      },
```

Comment:
According to the API docs, this is not an array of string but an array of objects

```json
        "notes": [
            {
                "author" : "Ann Doe",
                "content" : "I have followed up with John and we've scheduled a demo call to showcase our new features",
                "metricsType" : "NPS",
                "date" : "2020-06-28T04:35:15.482Z"
            },
        ],
```

Reply:
same here?
`"additionalQuestions": {
        "type": "array",
        "items": { "type": "object" }
      }